### PR TITLE
[Dialogs] Clean up swift test dependencies

### LINKED ISSUE #6214: [Dialogs] Add unit tests to bazel]
This was filed as an internal issue. If you are a Googler, please visit [b/122529930](http://b/122529930) for more details.

---
#### Internal data
- Associated internal bug: [b/122529930](http://b/122529930)

### PR DESCRIPTION
**Pull request by codeman7**
The unit test were also missing some imports, the missing imports were added.
Closes #6214

### CODE REVIEW

**Comment by romoore** [comment]
Looks like this may require an update to the `.kokoro` rewrite rules to build with bazel:

```
components/Dialogs/tests/unit/MDCAlertControllerAlertThemerTests.swift:18:8: error: no such module 'MaterialComponents.MDCAlertScheme'
import MaterialComponents.MDCAlertScheme
       ^
```

**Comment by andrewoverton** [comment]
https://github.com/material-components/material-components-ios/pull/5966 addresses this. I was planning to submit [cl/228493766](http://cl/228493766) today and then merge #5966 right after.

File: components/Dialogs/BUILD [hunk 1]
```diff
@@ -162,8 +162,13 @@ swift_library(
     ],
     visibility = ["//visibility:private"],
     deps = [
-        ":Theming",
+        ":Dialogs",
```
Comment (romoore) [inline components/Dialogs/BUILD]: Are these changes required? It looks like the test already exists after merging develop?  If they are, please update the PR title and description.

Comment (codeman7) [reply components/Dialogs/BUILD]: Yes, there are needed for the theming extension tests. The `BUILD` file had some dependencies but was missing the ones in _Beta_.

**Review by romoore — APPROVED** [review]
